### PR TITLE
Fix `glm_benchmark_analyze`

### PR DESCRIPTION
**Pull request by MartinStancsicsQC**
Checklist
* [x] Added a `CHANGELOG.rst` entry

Simple fix for #642

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,6 +7,14 @@
 Changelog
 =========
 
+2.5.2 - 2023-05-25
```
Comment (lbittarello) [inline CHANGELOG.rst:10]: Do we need a release just for this bug fix? It doesn't seem that urgent.

Comment (MartinStancsicsQC) [reply CHANGELOG.rst:10]: Agreed. Should I merge it with #641?

Comment (lbittarello) [reply CHANGELOG.rst:10]: No need – just change the date to "unreleased" (cf. #636). :)

Comment (MartinStancsicsQC) [reply CHANGELOG.rst]: Ah okay, thanks!

**Pull request merged**